### ISSUE TITLE
Feature 'resolved_breakpoints' does not have documented 'set'

### ISSUE THREAD

**Issue by KacerCZ** [xdebug/xdebug.org#104]
PR https://github.com/derickr/dbgp/pull/18 changed 'resolved_breakpoints' to opt-in feature but this change is not reflected at https://xdebug.org/docs/dbgp#feature-names

I am trying to get resolved breakpoints to work in NetBeans PHP debugger.
The only way how to get `resolved` attribute in `breakpoint_set` response was to call `feature_set -n resolved_breakpoints -v 1`.

**Comment by derickr** [comment]
The docs say that the **feature string** must always be available, not that resolved breakpoints must be available.

But it indeed does not include the `|set` that it should have. The reason why it is not enabled by default is because IDEs could choke on it if they see new XML packages that they don't understand. I'll come up with some better wording, although I think I have improved it in https://github.com/derickr/dbgp/blob/master/debugger_protocol.rst, which I have also just migrated to the xdebug.org website.
